src/ontology/cl-odk.yaml

```diff
@@ -32,6 +33,16 @@ subset_group:
     - id: eye_upper_slim
     - id: general_cell_types_upper_slim
     - id: kidney_upper_slim
+sssom_mappingset_group:
+  products:
+    - id: cl-local
+      maintainance: manual
+    - id: cl
+      maintainance: manual
```

Comment:
Seems the two mappings sets are not checked in - intended?

Reply:
> The FBbt and ZFA sets are committed to the repository (in src/mappings rather than src/ontology/mappings, to align CL with the recommendations of the ODK) since they are dependent on remote resources. This is the same logic as the one we use in Uberon.
>
> The CL-local and CL sets are not committed since they can always be re-generated as needed from files that are present in any copy of the repository.